dace/config_schema.yml

```diff
@@ -813,6 +813,12 @@ required:
         default: false
         title: Debug printing
         description: Enable verbose printouts.
+    
+    debugpass:
+        type: bool
+        default: false
+        title: Debug transformation pass
+        description: Enable saving SDFG before applying a (sub-)pass.
```

Comment:
I don't remember adding this but if I remove it I get a problem as `simplify.py` seems to use it.

Reply:
The usages where in other changes which I could undo